diamonds.js

```diff
@@ -0,0 +1,45 @@
+/* Write a program which has an array of diamonds with different value e.g. [1,2,3,4,5]. 
+The program should distribute the diamonds equally to 3 people. For example: Person 1: 1,4 Person 2: 2,3 Person 3: 5 
+There may be any number of diamonds with any value. If diamonds cannot be equally distributed(e.g [1,2,3,4]), 
+the program should print message “Cannot distribute equally to 3 people”. */
+var diamonds = [2,4,5,3,1,6];
+var total=0;
+var i;
+var remove;
+var pn=1;

```

Comment:
name properly, what is pn?

Reply:
person number

Reply:
so name it as personNumber, or currentPerson